slasher/rpc/detect_update_min_max_span.go

```diff
@@ -0,0 +1,103 @@
+package rpc
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pkg/errors"
+	ethpb "github.com/prysmaticlabs/prysm/proto/eth/v1alpha1"
+	"github.com/prysmaticlabs/prysm/shared/params"
+)
+
+type detect func(span uint64, spans *ethpb.MinMaxSpan, source uint64) uint64
+
+func detectMax(span uint64, spans *ethpb.MinMaxSpan, source uint64) uint64 {
```

Comment:
Would recommend putting unexported funcs at the bottom of the file if possible, especially if they don't have much context or are helper functions

Reply:
done